elements/pfe-button/BaseButton.scss

```diff
@@ -0,0 +1,488 @@
+}
+
+::slotted(button:active) {
+  --pf-c-button--m-link--Color: var(--pf-c-button--m-link--active--Color,
+    var(--pf-global--link--Color--hover, #004080));
+  --pf-c-button--m-link--BackgroundColor: var(--pf-c-button--m-link--active--BackgroundColor, transparent);
+}
+
+:host(:has(button:active)) {
+  --pf-c-button--m-link--Color: var(--pf-c-button--m-link--active--Color,
+    var(--pf-global--link--Color--hover, #004080));
+  --pf-c-button--m-link--BackgroundColor: var(--pf-c-button--m-link--active--BackgroundColor, transparent);
+}
+
+:host([variant=link][danger]) ::slotted(button) {
+  --pf-c-button--m-danger--Color: var(--pf-c-button--m-link--m-danger--Color,
+      var(--pf-global--danger-color--100, #c9190b));
+  --pf-c-button--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--BackgroundColor, transparent);
+}
+
+:host([variant=link][danger]) ::slotted(button:hover) {
+  --pf-c-button--m-link--m-danger--Color: var(--pf-c-button--m-link--m-danger--hover--Color,
+      var(--pf-global--danger-color--200, #a30000));
+  --pf-c-button--m-link--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--hover--BackgroundColor, transparent);
+}
+
+:host([variant=link][danger]) ::slotted(button:active) {
+  --pf-c-button--m-link--m-danger--Color: var(--pf-c-button--m-link--m-danger--active--Color,
+    var(--pf-global--danger-color--200, #a30000));
+  --pf-c-button--m-link--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--active--BackgroundColor, transparent);
+}
+
+:host([variant=link][danger]:has(button:active)) {
+  --pf-c-button--m-link--m-danger--Color: var(--pf-c-button--m-link--m-danger--active--Color,
+    var(--pf-global--danger-color--200, #a30000));
+  --pf-c-button--m-link--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--active--BackgroundColor, transparent);
+}
+
+:host([variant=link][danger]:is(:focus, :focus-within)) {
+  --pf-c-button--m-link--m-danger--Color: var(--pf-c-button--m-link--m-danger--focus--Color,
+    var(--pf-global--danger-color--200, #a30000));
+  --pf-c-button--m-link--m-danger--BackgroundColor: var(--pf-c-button--m-link--m-danger--focus--BackgroundColor, transparent);
+}
+
+/******************************
+ *                            *
+ *         DISABLED           *
+ *                            *
+ ******************************/
+
+:host([disabled]) #container ::slotted(button),
+#container ::slotted(button[aria-disabled=true]) {
+  pointer-events: none !important;
+  cursor: default !important;
+  color: var(--pf-c-button--disabled--Color,
+    var(--pf-global--disabled-color--100, #6a6e73)) !important;
+  background-color: var(--pf-c-button--disabled--BackgroundColor,
+    var(--pf-global--disabled-color--200, #d2d2d2)) !important;
+}
+
+:host([disabled]) #container ::slotted(button)::after,
+#container ::slotted(button[aria-disabled=true])::after {
+  border-color: var(--pf-c-button--disabled--after--BorderColor, transparent) !important;
+}
+
```

Comment:
The disabled link variants are still showing a background.  The suggestion below does not include CSS property stack as I wasn't sure what was appropriate here maybe `var(--pf-c-button--m-plain--disabled--BackgroundColor,  transparent);`?

```Suggestion

:host([disabled][variant="link"]) #container ::slotted(button) {
  background-color: transparent !important;
}

```